docs/CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ to secureblueadmin@proton.me
 ## I Want To Contribute
 
 > ### Legal Notice
-> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the project license.
+> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the project license. As such, contributing to this project using AI generated content of any kind is forbidden.
```

Comment:
I know I said to put this here but can we actually have it in a separate section. Cause copyright isn't the only reason to not have ai content, accuracy and quality is too.

maybe a separate section like:

### AI Code and Content Policy

AI generated content and code of any kind is forbidden.